docs/user-guide/install.md

```diff
@@ -158,3 +158,53 @@ Windows service vdvs uninstalled successfully!
 ```
 PS C:\> Set-ExecutionPolicy Unrestricted -Scope Process -Force
 ```
+
+## Upgrade Instructions
+
+The volume driver can be upgraded or reinstalled any time with no impact to the user data. User data in volumes existing before an upgrade of the volume driver is retained as it is and is accessible post an upgrade.
+
+An upgrade shouldn't have to always require un-install and install of both driver or plugin. At times only one may change.
+But it is recommended to stay on the latest same versions.
+The order of upgrade of driver or plugin doesn't matter.
+
+We do not require user to stop containers to do the driver or plugin upgrade.
+No change in behavior of vmgroup operations is expected due to driver or plugin upgrade.
+
+**ESX Driver upgrade:**
+1. Uninstall VIB (driver)
+```esxcli software vib remove -n esx-vmdkops-service```
+2. Download and install the latest VIB from the vsphere-storage-for-docker release page
```

Comment:
vsphere-storage-for-docker => docker-volume-vsphere :) (we are back to old name)